segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -124,6 +128,19 @@
     @Getter
     final private int lateWarningThresholdInMillis;
 
+    /**
+     * Minimum delay in seconds between when garbage chunks are marked for deletion and actually deleted.
+     */
+    @Getter
+    final private int garbageCollectionFrequencyInSeconds;
```

Comment:
Why isn't this a `Duration`? Then you don't have to worry about converting to seconds, millis, or whatever

Reply:
fixed